### PR TITLE
Replace abandoned package.

### PR DESCRIPTION
**Pull request by fletch3555**
I noticed a warning when running composer update that the zend package is abandoned and to use this one instead.  I didn't do any significant testing, so the version was a shot-in-the-dark.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "mockery/mockery": "^1.0",
         "squizlabs/php_codesniffer": "^3.3",
         "psalm/plugin-laravel": "^1.1",
-        "zendframework/zend-diactoros": "^1.0"
+        "laminas/laminas-diactoros": "^2.0"
```
Comment (ArlonAntonius) [inline composer.json]: For now, let's stay on `^1.0` and create a separate branch to test `^2.0` on an actual application before we actually change it.

**Comment by ArlonAntonius** [comment]
Thanks Fletcher :D
Once you fixed the small comment, can you get this PR out of draft? :)

**Review by ArlonAntonius — APPROVED** [review]
Completely fine, let's get this through ASAP 😛

File: composer.json (hunk 1 above)
Comment (fletch3555) [reply composer.json]: Done

**Pull request merged**